src/lib/mina_generators/user_command_generators.ml

```diff
@@ -37,15 +37,30 @@ let zkapp_command_with_ledger ?num_keypairs ?max_account_updates
     Option.value num_keypairs
       ~default:((max_account_updates * 2) + (max_token_updates * 3) + 2)
   in
-  let keypairs = List.init num_keypairs ~f:(fun _ -> Keypair.create ()) in
+  let new_keypairs =
+    let existing_keypairs =
+      ref
+      @@ Option.value_map ledger_init_state ~default:Keypair.Set.empty
+           ~f:(fun l ->
+             Array.map l ~f:(fun (kp, _balance, _nonce, _timing) -> kp)
+             |> Keypair.Set.of_array )
+    in
+    List.init num_keypairs ~f:(fun _ ->
+        let keypair = ref (Keypair.create ()) in
```

Comment:
Quickcheck generators are supposed to be deterministic, but `Keypair.create` will be different each time. Can you fix this so that the generator give the same output for the same input seed?